raiden/network/transport/matrix/utils.py

```diff
@@ -251,6 +251,22 @@ def _presence_listener(self, event: Dict[str, Any]):
         if self._user_presence_changed_callback:
             self._user_presence_changed_callback(user, new_state)
 
+    def get_status_info(self):
+        user_ids_to_presence_for_known_addresses = {
+            to_checksum_address(address): {
+                user_id: self._userid_to_presence[user_id].value
+                for user_id in self._address_to_userids[address]
+            }
+            for address in self.known_addresses
+        }
+
+        log.debug(
+            "Matrix status info",
```

Comment:
It's not really general status, maybe call it `Matrix presence info`?

Reply:
`Address mager info`? It also logs known addresses, user_ids.

Reply:
Yeah also fine.